contracts/wallet.sol

```diff
@@ -768,7 +769,8 @@ contract Wallet is ENSResolvable, Vault, GasTopUpLimit, LoadLimit {
     event UpdatedAvailableLimit();
 
     string constant public WALLET_VERSION = "3.1.0";
-    uint constant private _DEFAULT_MAX_STABLECOIN_LOAD_LIMIT = 10000; //10,000 USD
+    uint constant private _MIN_STABLECOIN_LOAD_LIMIT = 0;     // 1 USD
```

Comment:
I think this constant can go too, if we remove the check as I have described above.